rebar.config

```diff
@@ -25,7 +25,7 @@
 }.
 
 {plugins,
-  [{tx_xform, {git, "https://github.com/aeternity/tx_xform.git", {branch, "master"}}}]}.
+  [{tx_xform, {git, "https://github.com/zp-sd/tx_xform.git", {branch, "change-arity-of-aec_tx_new"}}}]}.
```

Comment:
See https://github.com/aeternity/tx_xform/pull/3.

Reply:
@zp-sd I would suggest you:
* Push branch change-arity-of-aec_tx_new in https://github.com/aeternity/tx_xform repo
* Make this rebar.config line refer to https://github.com/aeternity/tx_xform.git specific commit (rather than branch) i.e. `{ref, "fa96672"}` (referring to specific commit rather than master helps dealing with concurrent PRs as in this case)

Reply:
This depends on https://github.com/aeternity/epoch/pull/325 and https://github.com/aeternity/tx_xform/pull/3.